build-tools/automation/azure-pipelines.yaml

```diff
@@ -552,16 +552,16 @@ stages:
       parameters:
         artifactName: mac-bcl-test-results
 
-  # Check - "Xamarin.Android (Test MSBuild Mac)"
+  # Check - "Xamarin.Android (Test MSBuild - macOS)"
```

Comment:
The only thing I see to bring up is what the checks look like:

![image](https://user-images.githubusercontent.com/840039/65461012-bd6b0680-de18-11e9-975a-6039cdad27fa.png)

Since the end of the names get chopped off, is there a way to drop the name `Xamarin.Android` on the front? Is it just based on the `displayName`?

Reply:
I think we'd need to submit a feature request of sorts to the Azure Pipelines GitHub App team for this, unfortunately I'm not aware of a way to change the display names here. They seem to be constructed automatically based on pipeline name, stage name, and job name, and I haven't found any way to override that. Things are a _bit_ better on the Azure Pipelines website itself, and in the Checks summary on the main PR page though:

<img width="383" alt="Screen Shot 2019-09-23 at 4 49 34 PM" src="https://user-images.githubusercontent.com/2000163/65461625-391d8100-de22-11e9-849b-8f70be3285d5.png">

<img width="699" alt="Screen Shot 2019-09-23 at 4 51 04 PM" src="https://user-images.githubusercontent.com/2000163/65461758-69fdb600-de22-11e9-8d06-2fea8ba1f0e4.png">